app/helpers/application_helper/button/transform_vm_button.rb

```diff
@@ -0,0 +1,12 @@
+class ApplicationHelper::Button::TransformVmButton < ApplicationHelper::Button::Basic
+  needs :@record
+
+  def visible?
+    @record.vendor == "vmware"
```

Comment:
This looks wrong, shouldn't this be using some kind of `supports_*` (`SupportsFeatureMixin`)?

The idea is, we should never care about `vmware` vs `amazon`, only about "can you do X?".